lib/files/index.js

```diff
@@ -0,0 +1,120 @@
+const { mkdir, readdir, rm, rename, stat, writeFile } = require('fs/promises')
+const { existsSync }  = require('fs')
+const { dirname, join, normalize } = require('path')
+const multer = require('multer')
+
+const  filesInterface = (app, settings)  => {
+    const BASE_PATH = join(settings.rootDir, settings.userDir, 'storage')
+    // not sure about this yet, need to check how file storage works
+    const storage = multer.memoryStorage()
+    const upload = multer({storage: storage})
+
+    const regex = /^\/flowforge\/files\/_\/([\.a-zA-Z0-9\/\-_ ]*)$/
```

Comment:
We have to remember that a NR flow can create flows with any character in the filename - we cannot police what gets created. So the API needs to cope with anything; which may require url encoding certain entities.

Reply:
We can try `/^/flowforge\/files\/_\/(.*)$/` but that feels a little risky

Reply:
fixed, moved to encodeURIComponent to handle all options.